alf/config_util.py

```diff
@@ -576,6 +588,10 @@ def _decorate(fn_or_cls, name, whitelist, blacklist):
         blacklist (list[str]): A blacklisted set of kwargs that should not be
             configurable. All other kwargs will be configurable. Only one of
             ``whitelist` or ``blacklist`` should be specified.
+        guard_kwargs (list[str]): list of kwargs that should be guarded. In other
```

Comment:
The name `guard_kwargs` does really convey the meaning. How about  `config_only_args`?

Reply:
Good idea. Changed.